### PR TITLE
Reader: avoid unnecessary rerenders in following/manage/index.js

### PR DESCRIPTION
**Pull request by blowery**
Piece of the pie in preventing a re-render on every state change for following manage.

### CODE REVIEW

**Comment by samouri** [comment]
i've remove the other rerender issues by doing two things:
1. memoize getBlokeckedSites
2. move seed to a module level var

File: client/reader/following-manage/index.jsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ import { resemblesUrl, addSchemeIfMissing, withoutHttp } from 'lib/url';
 import { getReaderFollowsCount } from 'state/selectors';
 
 const PAGE_SIZE = 4;
+let RECS_SEED = random( 0, 10000 );
```
Comment (bluefuton) [inline client/reader/following-manage/index.jsx]: I'm not sure this should have constant naming if it can be changed later.

Comment (bluefuton) [inline client/reader/following-manage/index.jsx]: Fixed in b72d455
.

Comment (samouri) [inline client/reader/following-manage/index.jsx]: thanks @bluefuton, good catch

**Pull request merged**